mxnet-bot/PredictLabels/Predictor.py

```diff
@@ -41,15 +42,14 @@ def __init__(self):
         self.tv = None
         self.labels = None
         self.clf = None
-        self.reload()
 
-    def reload(self):
+    def reload(self, tv_file, clf_file, labels_file):
         """
         This method is to load models
         """
-        self.tv = pickle.load(open("/tmp/Vectorizer.p", "rb"))
-        self.labels = pickle.load(open("/tmp/Labels.p", "rb"))
-        self.clf = pickle.load(open("/tmp/Classifier.p", "rb"))
+        self.tv = pickle.load(open(tv_file, "rb"))
```

Comment:
Use "with open" idiom please, this leaks fds.

Reply:
Done. Replace all "open" with "with open",